MSSQL/preparePHSOntology.sql

```diff
@@ -72,4 +72,113 @@ GO
 create view i2o_ontology_lab as (select * from AllOfUs_Mart..i2b2metadata where i_stddomain='LOINC')
 GO
 create view i2o_ontology_drug as (select * from AllOfUs_Mart..pcornet_med where i_stddomain='RxNorm' or i_stddomain='NDC')
-GO
\ No newline at end of file
+GO
+
+--Manually UPSERT unit mappings for most commonly used unit values in the PHS XML values
```

Comment:
These seem useful even to people not using the XML-extraction above. Maybe they should go in a different script?

Reply:
This has already been separated out into it's own file.